test/test_grid.py

```diff
@@ -778,3 +781,57 @@ def test_build_face_edges_connectivity_fillvalues(self):
         self.assertTrue(
             np.array_equal(res_face_nodes_connectivity,
                            uds._ds["Mesh2_face_nodes"].values))
+
+
+class TestBallTree(TestCase):
+
+    corner_grid_files = [gridfile_CSne30, gridfile_mpas]
+    center_grid_files = [gridfile_mpas]
```

Comment:
Why don't you need a dataset here?

Reply:
Can you elaborate on what you mean?

Reply:
Yeah, so in line 788 you pass gridfile_CSne30, which if I remember is an integration variable of some type, like temperature or something similar, whereas in the center grid files you don't pass the same file. I was wondering why this was the case.

Reply:
gridfile_CSne30 is actually a grid file, there's also a corresponding data variable.